go/vt/vtgate/executor.go

```diff
@@ -1014,7 +1010,8 @@ func (e *Executor) handleMessageStream(ctx context.Context, safeSession *SafeSes
 		return vterrors.Errorf(vtrpcpb.Code_INTERNAL, "unrecognized STREAM statement: %v", sql)
 	}
 
-	table, err := vcursor.FindTable(streamStmt.Table)
+	// TODO: Ask Sugu about this??
+	table, _, err := vcursor.FindTable(streamStmt.Table)
```

Comment:
@sougou -  this looks like the right thing here, but wanted to double check with you.

Reply:
This is ok for now. Ideally, we want shard targeting to be honored here also. But can be changed later.